packages/core/src/libraries/hook.ts

```diff
@@ -15,6 +15,9 @@ import { LogEntry } from '#src/middleware/koa-audit-log.js';
 import type Queries from '#src/tenants/Queries.js';
 import { consoleLog } from '#src/utils/console.js';
 
+// Note: the retry limit is fixed to 3 for now.
```

Comment:
this comment is strange, since "for now" sounds "temporarily"

Reply:
didn't get the point though

Reply:
Removed.